src/components/RootApp/transformScalprumManifest.ts

```diff
@@ -11,13 +11,15 @@ function transformScalprumManifest(manifest: PluginManifest, config: ScalprumCon
       loadScripts: [],
     };
   }
+  let baseUrl = manifest.baseURL;
+  if (baseUrl === 'auto') {
```

Comment:
Actually, if we want to deliver this in commercial we would need to add `&& !manifest.cdnPath` check